shared/settings/delete-confirm/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react'
+import * as Styles from '../../styles'
+import * as Kb from '../../common-adapters'
+
+type CheckboxesProps = {
+  box1: boolean
```

Comment:
see this feedback: https://github.com/keybase/client/pull/19791#pullrequestreview-290642473

Reply:
@chrisnojima, mind taking another look?